### PR TITLE
PAY-5980: SPIKE: Duplicate PBA Payments -Idempotency Change

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
PAY-5980: SPIKE: Duplicate PBA Payments -Idempotency Change
         [PAY-5980] (https://tools.hmcts.net/jira/browse/PAY-5980).

### CODE REVIEW

**Review by davejones74 — APPROVED** [review]

File: charts/payment-api/values.preview.template.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ java:
     IAC_SERVICE_API_URL: http://ia-case-access-api-aat.service.core-compute-aat.internal
     PBA_CONFIG1_SERVICE_NAMES: dummy
     PCI_PAL_API_URL: https://ip3cloud.com/clients/hmcts/payments/index.php
-    REFUND_API_URL: http://ccpay-refunds-api-pr-370-java
+    REFUND_API_URL: http://ccpay-refunds-api-pr-381-java
```
Comment (Thor-tech-of-metal) [inline charts/payment-api/values.preview.template.yaml:22]: revert it back to master

File: dependency-check-suppressions.xml [hunk 2]
```diff
@@ -202,5 +204,18 @@
...
+        <notes>The fix for CVE-2023-24998 was incomplete for Apache Tomcat 11.0.0-M2 to 11.0.0-M4, 10.1.5 to 10.1.7, 9.0.71 to 9.0.73 and 8.5.85 to 8.5.87.
+        </notes>
+        <cve>CVE-2023-28709</cve>
+    </suppress>
+
+    <suppress>
+        <notes>CVE-2023-20883: Spring Boot Welcome Page DoS Vulnerability</notes>
+        <cve>CVE-2023-20883</cve>
+    </suppress>
+
```
Comment (Thor-tech-of-metal) [inline dependency-check-suppressions.xml:217]: revert it back to master

File: api/src/functionalTest/resources/application-functional-tests.properties [hunk 3]
```diff
@@ -48,9 +48,9 @@ launchDarkly.flushInterval=10
 pba.config1.service.names=${PBA_CONFIG1_SERVICE_NAMES:CMC}
 
 refund.api.url=${REFUND_API_URL:http://ccpay-refunds-api-aat.service.core-compute-aat.internal}
-bulk.scanning.payments.processed.url=${CCPAY_BULK_SCANNING_API_URL:http://ccpay-bulkscanning-api-aat.service.core-compute-aat.internal}
+# bulk.scanning.payments.processed.url=${CCPAY_BULK_SCANNING_API_URL:http://ccpay-bulkscanning-api-aat.service.core-compute-aat.internal}
 # refund.api.url=${REFUND_API_URL:https://ccpay-refunds-api-pr-370.preview.platform.hmcts.net}
-# bulk.scanning.payments.processed.url=${CCPAY_BULK_SCANNING_API_URL:https://ccpay-bulkscanning-api-pr-458.preview.platform.hmcts.net}
+bulk.scanning.payments.processed.url=${CCPAY_BULK_SCANNING_API_URL:https://ccpay-bulkscanning-api-pr-458.preview.platform.hmcts.net}
```
Comment (Thor-tech-of-metal) [inline api/src/functionalTest/resources/application-functional-tests.properties:54]: revert it back to master

**Review by Thor-tech-of-metal — COMMENTED** [review]
Comments needs to be resolved before merging.

**Pull request merged**